src/user-modules/tag-group.ts

```diff
@@ -116,34 +106,104 @@ class TagGroupMixin extends wechatifyMixinBase() {
     log.verbose('TagGroup', 'deleteTagGroup(%s)', tagGroup)
 
     try {
-      await this.wechaty.puppet.tagGroupDelete(tagGroup.id())
-      this.pool.splice(this.pool.indexOf(tagGroup), 1)
+      await this.wechaty.puppet.tagGroupDelete(tagGroup.id)
     } catch (e) {
       this.wechaty.emitError(e)
       log.error('TagGroup', 'deleteTagGroup() exception: %s', (e as Error).message)
     }
   }
 
   async tags (): Promise<TagInterface[]> {
-    return (await this.wechaty.Tag.list()).filter(tag => tag.groupId() === this.id())
+    const tagList = await this.wechaty.Tag.list()
```

Comment:
Why do not use `tagGroupTagList()` which already defined in `wechaty-puppet`?

```ts
await this.wechaty.Tag.tagGroupTagList(this.id)
```

Reply:
Cause I fogot that...